website/static/js/keen.js

```diff
@@ -0,0 +1,143 @@
+var keen = require('keen-js');
```

Comment:
I believe jquery is available on every page through webpack and doesn't need to be required when used.